package.json

```diff
@@ -22,6 +22,7 @@
         "rxjs": "6.4.0",
         "systemjs": "0.21.6",
         "tslib": "1.9.3",
+        "tslint": "^5.15.0",
```

Comment:
why are you adding tslint?

Reply:
at some point, I was trying to see if I can use the MD schematics eg, `ng generate @angular/material:table foo`. But this command was requiring `tslint`.

I will remove this dependency, now it is needed anymore.